README.md

```diff
@@ -26,7 +26,7 @@ Querylist can be installed like any other python package:
 
     > pip install querylist
 
-Querylist is tested against Python 2.6, 2.7, 3.3, 3.4, and pypy.
+Querylist is tested against Python 3.6+ and pypy.
```

Comment:
There might be duplicate content in docs/ that needs updating.

Reply:
Thanks, updated docs too. Also, I allowed myself to update CHANGELOG.